core/src/main/resources/reference.conf

```diff
@@ -537,6 +537,12 @@ cassandra-query-journal {
 
   # New events are retrieved (polled) with this interval.
   refresh-interval = 3s
+  
+  # Sequence numbers for a persistenceId is assumed to be monotonically increasing
+  # without gaps. That is used for detecting missing events.
+  # In early versions of the journal that might not be true and therefore
```

Comment:
Does "in earlier versions" mean "journals, that were produced by earlier versions of this persistence plugin"?

Reply:
yes, there is risk. For example I think we supported rejections previously (serialization failed) and then the persistent actor could happily continue with next seq nr and there would be a gap for the rejected